js/views/endScreen.js

```diff
@@ -37,6 +38,8 @@ var EndScreen = React.createClass({
 
         <a className="oo-state-screen-selectable" onClick={this.handleClick}></a>
```

Comment:
we can also remove all `oo-state-screen-selectable` that are children of `oo-interactive-container`.

now that users can click through `oo-interactive-container` we don't need duplicate `oo-state-screen-selectable`